source/FingerBoard-Core/FBNodeStringEvent.class.st

```diff
@@ -0,0 +1,31 @@
+Class {
+	#name : #FBNodeStringEvent,
```

Comment:
@noha I renamed the event class
Also, this is a subclass of FBNodeActivationEvent, and can keep a reference to the activation.

Reply:
Hmm, If it is a string event it should be basic and called FBStringEvent, no? Holding an activation and a string does not make sense to me. And if so the node activation event can hold a string as well although it will create that from the activation and another object